### DISCUSSION TITLE
gcp/2020/05/09/google-cloud-build/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#154] [Comments]
# gcp/2020/05/09/google-cloud-build/

 Google Cloud Platform의 Cloud Build 서비스를 사용하는 방법에 대해 정리한 글입니다 

https://zzsza.github.io/gcp/2020/05/09/google-cloud-build/

**Comment by jae-heo** [comment]
요즘 인프라 공부를 하고있는데 관련해서 포스팅 많이 올려주셔서 큰 도움 되고 있습니다.
감사합니다.
